### PR TITLE
Package creation for Beets music organizer (v1.4.3)

### PR DESCRIPTION
**Pull request by ymartin59**
- Request #2572
- Follow wheels distribution guidelines
- Detailled installation log

_Linked issues:_ #2572 

### Checklist
- [X] Build rule `all-supported` completed successfully
- [X] New installation of package completed successfully. Tested on XPEnologoy DS3615xs `5.2-5967.1`

### CODE REVIEW

**Comment by cytec** [comment]
LGTM

File: spk/beets/Makefile [hunk 1]
```diff
@@ -0,0 +1,26 @@
+SPK_NAME = beets
+SPK_VERS = 1.4.3
+SPK_REV = 1
+SPK_ICON = src/beets.png
+
+WHEELS = src/requirements.txt
+SPK_DEPENDS = "python>=2.7.8-11"
+
+MAINTAINER = Volunteer?
```
Comment (Dr-Bean) [inline spk/beets/Makefile]: You are the maintainer, or at the very least, the creator of the package...although I hope you can keep an eye on the upstream version and update it every once in a while.

File: spk/beets/src/installer.sh [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    # Link
+    ln -s ${SYNOPKG_PKGDEST} ${INSTALL_DIR}
+
+    # Create a Python virtualenv
+    ${VIRTUALENV} --system-site-packages ${INSTALL_DIR}/env > ${INSTALL_DIR}/install-details.log
+
+    # Install the wheels
+    ${INSTALL_DIR}/env/bin/pip install --no-deps --no-index -U --force-reinstall -f ${INSTALL_DIR}/share/wheelhouse ${INSTALL_DIR}/share/wheelhouse/*.whl > ${INSTALL_DIR}/install-details.log 2>&1
+
+    # Log installation details
```
Comment (Dr-Bean) [inline spk/beets/src/installer.sh]: To be fair, this looks kind of messy to me. Let's leave it for now, so you can gather feedback for the time the package is in beta, but let's remove it (or a cleanup) when the beta tag is removed. Applies to this section, redirect on L26, redirect on L29.
Agree?

**Comment by Dr-Bean** [comment]
Two comments, one of which has to be changed before merging (maintainer). Otherwise GTG, nice work :)

File: spk/beets/Makefile (hunk 1 above)
Comment (ymartin59) [inline spk/beets/Makefile]: As I expect to use that tool myself... Let's say I am maintainer.

**Comment by Dr-Bean** [comment]
Published.
@ymartin59 I'll probably change your name as maintainer to your Github username, or you can open a PR for that. While not implemented (yet, I suppose), the plan was to link that together with the `Author` field on synocommunity.com, and that in turn to your Github profile...or something, it's been a while since it was discussed.

Anyway, go ahead and register on https://synocommunity.com/register if you want to get credited as author on `synocommunity.com/package/beets` instead of me or someone else.
The same applies to @o0v7r0o for `synocommunity.com/package/syncthing-inotify`, @ressu for `synocommunity.com/package/radarr`, and @adrien-delhorme for synocommunity.com/package/duplicity while I'm at it.

**Pull request merged**